File: 프로그래머스/unrated/181912. 배열 만들기 5/배열 만들기 5.java
import java.util.ArrayList;
class Solution {
    public int[] solution(String[] intStrs, int k, int s, int l) {
        ArrayList<Integer> answerList = new ArrayList<>();
        int idx =0;
        for(int i = 0; i<intStrs.length; i++){
            int num = Integer.parseInt(intStrs[i].substring(s, s + l));
                if (num > k) {
                    answerList.add(num);
                }
        }
        // ArrayList를 배열로 변환
        int[] answer = new int[answerList.size()];
        for (int i = 0; i < answerList.size(); i++) {
            answer[i] = answerList.get(i);
        }
        return answer;
    }
}

File: 프로그래머스/0/181912. 배열 만들기 5/배열 만들기 5.js
function solution(intStrs, k, s, l) {
    var answer = [];
    for (let i of intStrs) {
        let num = parseInt(i.slice(s, s + l), 10);
        if (num > k) {
            answer.push(num);
        }
    }
    return answer;
}
